Samples/Desktop/D3D12Raytracing/tools/x64/PostbuildCopy.bat

```diff
@@ -0,0 +1,23 @@
+rem Get the Windows 10 version and trim everything off except the number itself
```

Comment:
@pkristof Can you try this on an RS4 machine and verify that it grabs the D3D12 binary?